tests/lax_numpy_test.py

```diff
@@ -1152,6 +1152,22 @@ def testPolySub(self, a_shape, b_shape, dtype):
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "a_shape={} , order={}".format(
+          jtu.format_shape_dtype_string(a_shape, dtype),
+          order),
+       "dtype": dtype, "a_shape": a_shape, "order" : order}
+      for dtype in default_dtypes
+      for a_shape in one_dim_array_shapes
+      for order in range(8)))
+  def testPolyDer(self, a_shape, order, dtype):
+    rng = jtu.rand_default(self.rng())
+    np_fun = lambda arg1, arg2: np.polyder(arg1, m=arg2)
+    jnp_fun = lambda arg1, arg2: jnp.polyder(arg1, m=arg2)
+    args_maker = lambda: [rng(a_shape, dtype), order]
+    self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=False)
+
```

Comment:
We should add a `CompileAndCheck` here. I believe it should pass once the above arg comment is addressed